tests/test_serializers.py

```diff
@@ -707,11 +707,14 @@ def test_get_all_fields(self):
         home_field_1 = self.serializer.fields['home']
         home_field_2 = all_fields['home']
 
+        '''
+        # Expected with fields cache
```

Comment:
maybe remove this block, or wrap in check to only run if fields caching is disabled.